hpx/runtime/components/server/migration_support.hpp

```diff
@@ -239,5 +243,4 @@ namespace hpx { namespace components
         hpx::lcos::local::promise<void> trigger_migration_;
         bool was_marked_for_migration_;
     };
-}}
-
+}}    // namespace hpx::components
```

Comment:
Same here: should this be formatted now?

Reply:
No, this was accidental. I'll stick to only formatting/sorting includes in these files.

Reply:
Removed these too.